wombat/src/main/include/utils/Encoder.h

```diff
@@ -31,7 +31,7 @@ namespace wom {
     units::radians_per_second_t GetEncoderAngularVelocity();   // rad/s
 
     int encoderType = 0;
-    double _reduction = 1.0;
+    double _reduction;
```

Comment:
not allowing a default reduction?

Reply:
It didnt want it here as this initialises it before the constructor assigns it from the param, but if I put the default in the constructor it wants a default for everything

Reply:
do you want me to add it as an issue and you can solve it another time?

Reply:
sounds good :+1: